<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginToolViewWidget</class>
 <widget class="QWidget" name="PluginToolViewWidget">
  <property name="windowTitle">
   <string>C++ Helper ToolView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="diagnosticTab">
      <attribute name="title">
       <string>Diagnostic Messages</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="diagnosticMessages">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Monospace&quot;;</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="includesTab">
      <attribute name="title">
       <string>#include Explorer</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_3_0">
       <item>
        <widget class="QSplitter" name="includesSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="lw_3_1">
          <layout class="QVBoxLayout" name="vl_3_2">
           <item>
            <widget class="QLabel" name="treeLabel">
             <property name="text">
              <string>Include files tree:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="includesTree">
             <property name="toolTip">
              <string/>
             </property>
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;#include&lt;/span&gt; files used by current translation unit. Redundand &lt;tt&gt;#includes&lt;/tt&gt; printed in yellow color.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="lw_3_2">
          <layout class="QVBoxLayout" name="vl_3_3">
           <item>
            <widget class="QLabel" name="includedByLabel">
             <property name="text">
              <string>Included by:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="includedFromList">
             <property name="toolTip">
              <string>Double click to open corresponding document</string>
             </property>
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of files which &lt;tt&gt;#include&lt;/tt&gt; a selected one&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl_3_1">
         <item>
          <widget class="KTreeWidgetSearchLine" name="searchFilter">
           <property name="toolTip">
            <string>Quick find in a tree</string>
           </property>
           <property name="placeholderText">
            <string>Quick search for file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="updateButton">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Update &lt;tt&gt;#include&lt;/tt&gt; files tree for current document&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="searchTab">
      <attribute name="title">
       <string>Search</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_2_0">
       <item>
        <widget class="QSplitter" name="searchSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="lw_2_1">
          <layout class="QVBoxLayout" name="vl_2_1">
           <item>
            <widget class="KLineEdit" name="searchQuery">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search query may contain names and/or special terms:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;access:public|protected|private, anon:y&lt;/span&gt; or &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;anonymous:y, base:&amp;lt;class-name&amp;gt;, inheritance:public|protected|private, decl:&amp;lt;name&amp;gt;, ref:&amp;lt;name&amp;gt;, kind:&amp;lt;value&amp;gt;, pod:y, scope:&amp;lt;name&amp;gt;, static:y, virtual:y, arity:M..N, align:M..N, size:M..N.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Valid values for &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;kind:&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;ns&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;ns-alias&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;typedef&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,
courier';&quot;&gt;type-alias&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;struct&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;class&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;union&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;enum&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;enum-const&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;fn&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;method&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;ctor&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;dtor&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;conversion&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;param&lt;/span&gt;, &lt;span style=&quot; font-
family:'Courier New,courier';&quot;&gt;var&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;field&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;bitfield.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Search terms can be combined into a logical expressions with the following keywords: &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;AND&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;OR&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;NOT&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New,courier';&quot;&gt;XOR&lt;/span&gt; and/or parentheses.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Enter search query</string>
             </property>
             <property name="showClearButton" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeView" name="searchResults">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="itemsExpandable">
              <bool>false</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="expandsOnDoubleClick">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="details">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Symbol Details</string>
          </property>
          <layout class="QFormLayout" name="detailsFormLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <property name="rowWrapPolicy">
            <enum>QFormLayout::WrapLongRows</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="indexerSettingsTab">
      <attribute name="title">
       <string>Indexer Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QSplitter" name="indicesSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="lw_4_1">
          <layout class="QVBoxLayout" name="vl_4_1">
           <item>
            <widget class="QLabel" name="databasesLabel">
             <property name="text">
              <string>Databases:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeView" name="databases">
             <property name="toolTip">
              <string>List of available index collections</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="itemsExpandable">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="expandsOnDoubleClick">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hl_4_1">
             <item>
              <widget class="KPushButton" name="newDatabase">
               <property name="toolTip">
                <string>Add new index collection</string>
               </property>
               <property name="text">
                <string>New</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KPushButton" name="deleteDatabase">
               <property name="toolTip">
                <string>Remove selected index</string>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="sp_4_1">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="KPushButton" name="reindexDatabase">
               <property name="toolTip">
                <string>Start rebuilding selected index</string>
               </property>
               <property name="text">
                <string>Reindex</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KPushButton" name="stopIndexer">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Stop indexing</string>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="lw_4_2">
          <layout class="QVBoxLayout" name="vl_4_3">
           <item>
            <widget class="QGroupBox" name="indexingOptions">
             <property name="title">
              <string>Indexing Options</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCheckBox" name="indexFunctionBody">
                <property name="text">
                 <string>Index function local symbols</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="skipImplicits">
                <property name="text">
                 <string>Skip implicit template instantiations</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="targetsLabel">
             <property name="text">
              <string>Index Targets:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="targets">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>List of targets to be indexed</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hl_4_2">
             <item>
              <spacer name="sp_4_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="KPushButton" name="addTarget">
               <property name="toolTip">
                <string>Add new target to be indexed</string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KPushButton" name="removeTarget">
               <property name="toolTip">
                <string>Remove index target</string>
               </property>
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
